numpy/core/src/umath/override.c

```diff
@@ -495,104 +240,58 @@ PyUFunc_CheckOverride(PyUFuncObject *ufunc, char *method,
     }
 
     /*
-     * Normalize ufunc arguments.
+     * Normalize ufunc arguments, note that args does not hold any positional
```

Comment:
This note was very confusing to me: maybe "note that any input and output arrays in args have already been stored in `in_args` and `out_args`"

The comment about `len_args` seems false (at least for reductions).

Reply:
Hmmmpf, this probably made a bit more sense in the old (more complicated) iteration.  Adopting your comment.